racket/collects/racket/splicing.rkt

```diff
@@ -343,9 +343,14 @@
                (syntax/loc/props body
                  (begin-for-syntax (wrap-param-et e (orig-id ...) (temp-id ...)) ...))]
               [(module . _) body]
-              [(module* n #f form ...)
-               (syntax/loc/props body
-                 (module* n #f (expand-ssp-body (sp-id ...) (temp-id ...) (orig-id ...) form) ...))]
+              [(module* name #f form ...)
+               (with-syntax ([#%module-begin (datum->syntax body '#%module-begin)])
```

Comment:
Is this binding used? I think it would have to appear syntactically in the `with-syntax` body to be relevant.

Reply:
Whoops, you’re right—that’s vestigial from an older version of the code. I’ll remove it.